config/config_notwin_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The Prometheus Authors
```

Comment:
Got it, sorry you are right. I just don't like the `notwin` in the name. We could do `config_default_test.go` `config_win_test.go` like here: https://golang.org/src/cmd/dist/ (see sys_*).